dcs/__init__.py

```diff
@@ -20,5 +20,6 @@
 from . import forcedoptions
 from . import installation
 from . import nav_target_point
+from . import password
```

Comment:
not sure if we need that in the main package domain?
I think it would be enough to just import it in `unitgroup.py`

Reply:
@rp- the reason I added password as a separate module is due to the fact that I'm working on another patch which adds support for ground control passwords. And it felt wrong to import it from unitgroup. If you think it's better to keep it in unitgroup, I can definitely import it that way.

Reply:
But I guess the password module will just used internally?
So there will be no reason to "export" the module in the main module.

Reply:
@rp- noted. Updated.